explorer/client/src/components/transaction-card/RecentTxCard.tsx

```diff
@@ -45,58 +45,47 @@ type TxnData = {
 async function getRecentTransactions(txNum: number): Promise<TxnData[]> {
     try {
```

Comment:
Can we handle the error correctly? line 94 just throws the error. Let's make sure it's getting logged and show something like `There is an error fetching recent transactions` on the UI

Reply:
i removed the original step that did this because it seemed like we had redundant steps for errors in the try / catch blocks

Reply:
I added the `setIsLoaded(false)` line when any error happens so that the UI will display the error message we already had:
commit: cee7030d598b002ca67e2277240215bab17d35ae